recipes/fire-hpp/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${CMAKE_PROJECT_NAME} test_package.cpp)
+target_link_libraries(${CMAKE_PROJECT_NAME} ${CONAN_LIBS})
+set_property(TARGET ${CMAKE_PROJECT_NAME} PROPERTY CXX_STANDARD 14)
```

I removed the `CXX_STANDARD 14` because its [cmake build script](https://github.com/kongaskristjan/fire-hpp/blob/master/CMakeLists.txt) claims it is c++11.
Also test the `cmake_find_package`.

Unless I'm wrong of course.

```suggestion
conan_basic_setup(TARGETS)

find_package(fire-hpp REQUIRED CONFIG)

add_executable(${CMAKE_PROJECT_NAME} test_package.cpp)
target_link_libraries(${CMAKE_PROJECT_NAME} PRIVATE fire-hpp::fire-hpp)
```